GameContent/GameHandler.cs

```diff
@@ -238,6 +240,32 @@ internal static void DoRender()
                 }
             }
 
+            List<UIElement> trySlider = UIElement.GetElementAt(GameUtils.MousePosition);
+
+            if (trySlider.Count > 0)
+            {
+                if (trySlider[0] != null)
+                {
+                    UIElement elementWeWant = trySlider[0].GetElementAt(GameUtils.MousePosition);
+                    if (elementWeWant.GetType() != typeof(UIImage))
+                        return;
+
+                    elementWeWant.LeftClick();
+                    elementWeWant.LeftDown();
+                    elementWeWant.LeftUp();
+
+                    elementWeWant.RightClick();
+                    elementWeWant.RightDown();
+                    elementWeWant.RightUp();
+
+                    elementWeWant.MiddleClick();
+                    elementWeWant.MiddleDown();
+                    elementWeWant.MiddleUp();
+
+                    elementWeWant.MouseOver();
+                } 
+            }
```

Comment:
Don't think this is the right place to update it. I'd recommend making a method like `UIElement.Update()`?